hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/SCMNodeManager.java

```diff
@@ -540,37 +540,35 @@ public void processLayoutVersionReport(DatanodeDetails datanodeDetails,
           layoutVersionReport.toString().replaceAll("\n", "\\\\n"));
     }
 
-    if (layoutVersionReport != null) {
-      int scmSlv = scmLayoutVersionManager.getSoftwareLayoutVersion();
-      int scmMlv = scmLayoutVersionManager.getMetadataLayoutVersion();
-      int dnSlv = layoutVersionReport.getSoftwareLayoutVersion();
-      int dnMlv = layoutVersionReport.getMetadataLayoutVersion();
-
-      // If the data node slv is > scm slv => log error condition
-      if (dnSlv > scmSlv) {
-        LOG.error("Rogue data node in the cluster : {}. " +
-                "DataNode SoftwareLayoutVersion = {}, SCM " +
-                "SoftwareLayoutVersion = {}",
-            datanodeDetails.getHostName(), dnSlv, scmSlv);
-      }
+    int scmSlv = scmLayoutVersionManager.getSoftwareLayoutVersion();
+    int scmMlv = scmLayoutVersionManager.getMetadataLayoutVersion();
+    int dnSlv = layoutVersionReport.getSoftwareLayoutVersion();
+    int dnMlv = layoutVersionReport.getMetadataLayoutVersion();
+
+    // If the data node slv is > scm slv => log error condition
+    if (dnSlv > scmSlv) {
+      LOG.error("Rogue data node in the cluster : {}. " +
```

Comment:
Rouge is generally used in the security context, I'd suggest using Outdated or something similar.